pyproject.toml

```diff
@@ -0,0 +1,57 @@
+# --- PROJECT CONFIGURATION --- #
+
+[build-system]
+requires = ["setuptools>=67", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "forecast_blend"
+dynamic = ["version"]  # Automatically manage versioning
+description = "This app blends forecasts together using weights and saves them for API use."
+readme = "README.md"
+requires-python = ">=3.8"
+license = {text = "MIT License"}
+authors = [
+    { name = "Rishi Prasad", email = "rishiprasad002@gmail.com" }
```

Comment:
sorry, could you take your name out, and just write Open Climate Fix,with info@openclimatefix.org